core/rio/turtle/src/main/java/org/eclipse/rdf4j/rio/turtle/ArrangedWriter.java

```diff
@@ -205,29 +203,27 @@ protected synchronized void consumeStatement(Statement st) throws RDFHandlerExce
 	}
 
 	private synchronized Statement nextStatement() {
-
-		if (statementBySubject.isEmpty() && blanks.isEmpty()) {
+		if (stmtBySubject.isEmpty() && blanks.isEmpty()) {
 			assert queueSize == 0;
 			return null;
 		}
-		Set<Statement> statements = null;
-		while (statements == null) {
+		Set<Statement> stmts = null;
+		while (stmts == null) {
```

Comment:
All the changes in this file are just the result of me reverting some earlier attempts to fix the inlining issues. They're not really relevant as the ArrangedWriter class is no longer used and only left in place on the off-chance that some third party uses it (unlikely but you never know).